docs/setup_vscode.md

```diff
@@ -480,189 +414,144 @@ To configure input redirection, edit `launch.json`.
-
-# Pro-tips
-The C/C++ extension for Visual Studio Code has more features.  Check out this Blog Post from the Visual C++ Team Blog.  [https://blogs.msdn.microsoft.com/vcblog/2016/03/31/cc-extension-for-visual-studio-code/](https://blogs.msdn.microsoft.com/vcblog/2016/03/31/cc-extension-for-visual-studio-code/)
-
-You can optionally use the command line to install VS Code extensions.
-```console
-$ code --install-extension ms-vscode.cpptools
-Found 'ms-vscode.cpptools' in the marketplace.
-Installing...
-Extension 'ms-vscode.cpptools' v0.14.0 was successfully installed!
-```
-
-
-# Next steps
-[Return to the main set up tutorial.](setup.html#visual-debugger)
+<div class="primer-spec-callout warning" markdown="1">
+If you have trouble viewing the contents of a container like this screenshot, see [Pretty Printing STL Containers with `gdb`](setup_gdb.html#pretty-printing-stl-containers-with-gdb).
 
+<img src="images/vscode140.png" width="480px" />
+</div>
 
-# Troubleshooting & FAQ
+### Step into
+Click "Step Into".  The cursor enters the `sum()` function.
 
-## Pretty-printing STL Containers with `gdb`
-Some versions of Visual Studio Code uses `gdb` under the hood.  Some installations of `gdb` don't ship with pretty printing support for STL containers like `vector`.  If you have this problem, you'll see something like this when you try to view the contents of a `vector` while debugging.  See [Pretty Printing STL Containers with `gdb`](setup_gdb.html#pretty-printing-stl-containers-with-gdb).
+<img src="images/vscode115.png" width="768px" />
 
-<img src="images/vscode140.png" width="480px" />
+<img src="images/vscode120.png" width="768px" />
 
+### Step out
+Click "Step Out".  The `sum()` function completes, and the program pauses again.
 
-## macOS Visual Debugger Issues
-On older versions of macOS, VS Code, or the C/C++ extension, you may experience issues with the visual debugger. Please try the following steps to troubleshoot your debugger:
+<img src="images/vscode125.png" width="768px" />
 
-1. Restart your computer
-2. Update macOS, VS Code, and the C/C++ extension.  This tutorial has been tested as of macOS Big Sur 11.1, VS Code 1.52.1, and C/C++ Extension version 1.1.3.
-3. Try using CodeLLDB's debugger instead (see below)
+<img src="images/vscode130.png" width="768px" />
 
-### Install CodeLLDB
-{: .primer-spec-toc-ignore }
-Another option in troubleshooting your visual debugger is to replace the one that comes with the C/C++ extension you set up in the beginning of this tutorial.
+### Continue
+Press "Continue" to run the program to the next breakpoint, or the end, whichever comes first.
 
-We **do not** recommend you uninstall the C/C++ extension because it provides syntax highlighting and other various niceties; instead, you can have both plugins installed concurrently.
+<img src="images/vscode135.png" width="768px" />
 
```

Comment:
Perhaps a final "summary" image of the debugging pane (the 6 icons), with each icon labelled by its name. Think it will be useful for students to be able to see all the labels at once.

Reply:
Are you thinking like [this](https://preview.sesh.rs/previews/eecs280staff/tutorials/27/debugging_guide.html#basic-controls) from PR https://github.com/eecs280staff/tutorials/pull/27 ?

Reply:
Yes exactly!